src/handleIncomingRequest.ts

```diff
@@ -40,7 +54,7 @@ export const createIncomingRequestHandler = (
 
     const mockedResponse:
       | MockedResponse
-      | undefined = relevantRequestHandler.resolver(
+      | undefined = await relevantRequestHandler.resolver(
```

Comment:
Response resolvers are now async, allowing to perform async side-effects inside them, such as `ctx.fetch()`.